bundles/net.tourbook/src/net/tourbook/ui/views/TourNutritionView.java

```diff
@@ -0,0 +1,1628 @@
+                  .hint(HINT_TEXT_COLUMN_WIDTH, SWT.DEFAULT)
+                  .align(SWT.END, SWT.FILL)
+                  .applyTo(_txtSodium_Total);
+
+            // Unit: mg
+            UI.createLabel(container, UI.UNIT_WEIGHT_MG);
+         }
+
+         /*
+          * Averages
+          */
+         {
+            final Label label = UI.createLabel(container, Messages.Tour_Nutrition_Label_Averages, Messages.Tour_Nutrition_Label_Averages_Tooltip);
+            GridDataFactory.fillDefaults().align(SWT.CENTER, SWT.CENTER).applyTo(label);
+
+            _txtCalories_Average = _tk.createText(container, UI.EMPTY_STRING, SWT.TRAIL);
+            _txtCalories_Average.setEnabled(false);
+            GridDataFactory.fillDefaults().hint(HINT_TEXT_COLUMN_WIDTH, SWT.DEFAULT).align(SWT.END, SWT.FILL)
+                  .applyTo(_txtCalories_Average);
+
+            // Unit: kcal/h
+            UI.createLabel(container, OtherMessages.VALUE_UNIT_K_CALORIES + UI.SLASH + UI.UNIT_LABEL_TIME);
+
+            _txtFluid_Average = _tk.createText(container, UI.EMPTY_STRING, SWT.TRAIL);
+            _txtFluid_Average.setEnabled(false);
+            GridDataFactory.fillDefaults().hint(HINT_TEXT_COLUMN_WIDTH, SWT.DEFAULT).align(SWT.END, SWT.FILL)
+                  .applyTo(_txtFluid_Average);
+
+            // Unit: L/h
+            UI.createLabel(container, UI.UNIT_FLUIDS_L + UI.SLASH + UI.UNIT_LABEL_TIME);
+
+            _txtSodium_Average = _tk.createText(container, UI.EMPTY_STRING, SWT.TRAIL);
+            _txtSodium_Average.setEnabled(false);
+            GridDataFactory.fillDefaults().hint(HINT_TEXT_COLUMN_WIDTH, SWT.DEFAULT).align(SWT.END, SWT.FILL)
+                  .applyTo(_txtSodium_Average);
+
+            // Unit: mg/h
+            UI.createLabel(container, UI.UNIT_WEIGHT_MG + UI.SLASH + UI.UNIT_LABEL_TIME);
+         }
+      }
+   }
+
+   private void createUI_210_Actions(final Composite parent) {
+
+      /*
+       * Search product button
+       */
+      final Button btnSearchProduct = new Button(parent, SWT.NONE);
+      btnSearchProduct.setText(Messages.Tour_Nutrition_Button_SearchProduct);
+      btnSearchProduct.setToolTipText(Messages.Tour_Nutrition_Button_SearchProduct_Tooltip);
+      btnSearchProduct.addSelectionListener(widgetSelectedAdapter(selectionEvent -> new DialogSearchProduct(Display.getCurrent().getActiveShell(),
+            _tourData.getTourId()).open()));
+      btnSearchProduct.setImage(_imageSearch);
+      GridDataFactory.fillDefaults().align(SWT.BEGINNING, SWT.CENTER).grab(true, false).applyTo(btnSearchProduct);
+
+      /*
+       * Add product button
+       */
+      final Button btnAddCustomProduct = new Button(parent, SWT.NONE);
+      btnAddCustomProduct.setText(Messages.Tour_Nutrition_Button_AddCustomProduct);
+      btnAddCustomProduct.setToolTipText(Messages.Tour_Nutrition_Button_AddCustomProduct_Tooltip);
+      btnAddCustomProduct.addSelectionListener(widgetSelectedAdapter(selectionEvent -> {
```

Comment:
How can a custom product be edited?

Reply:
A custom product cannot be edited at the moment, but it could be a future improvement